Assets/LeapMotion/Scripts/DetectionUtilities/DetectorLogicGate.cs

```diff
@@ -0,0 +1,79 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using System.Collections.Generic;
+
+namespace Leap.Unity {
+
+  public class DetectorLogicGate : Detector {
+    public List<Detector> Detectors;
```

Comment:
This list should not be public, since modification would cause incorrect behaviour.